packages/presentation/presentation_publishing/publishing_subject/publishing_batcher.ts

```diff
@@ -46,17 +51,20 @@ export const useBatchedPublishingSubjects = <SubjectsType extends [...AnyPublish
   /**
    * Set up latest published values state, initialized with the current values of the subjects.
    */
-  const initialSubjectValues = useMemo(
-    () => unwrapPublishingSubjectArray(subjectsToUse),
-    [subjectsToUse]
-  );
+  const subjectValues = useMemo(() => unwrapPublishingSubjectArray(subjectsToUse), [subjectsToUse]);
```

Comment:
nit: this `useMemo` isn't needed if it's only used to initialize the `useState` hook on line 56. The initializer should only get run once anyway.

Reply:
I removed useMemo and just moved the function call into useState method.